version.sbt

```diff
@@ -1 +1 @@
-version in ThisBuild := "0.1.16-SNAPSHOT"
\ No newline at end of file
+version in ThisBuild := "0.2.0-SNAPSHOT"
```

Comment:
version number is automatically bumped when publishing to maven. don't change this file manually please.

Reply:
Ah, thank you! Wasn't sure if auto-incrementing was set up for the project. Will change this back now.